Twittimation/Commands/ScheduleLike.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using Twittimation.IO;
+
+namespace Twittimation.Commands
+{
+    public sealed class ScheduleLike : Command
+    {
+        public override List<string> RequiredArgs { get; } = new List<string>() { "Time", "TweetId" };
+        public override List<string> OptionalArgs { get; } = new List<string>();
+        public override Optional<string> OptionalRepeatedArg { get; } = new Optional<string>();
+        public override string HelpInfo { get; } = "Likes the specified tweet at the scheduled time.";
+        public override string ExtendedHelp { get; } = "Likes the specified tweet at the scheduled time. \r\nTime can be formatted as \"Year/Month/Day Hour:Minute:Second\".";
+
+        private readonly IStored<Tasks> _tasks;
+        private readonly Like _like;
+
+        public ScheduleLike(IStored<Tasks> tasks, Like like)
+        {
+            _tasks = tasks;
+            _like = like;
+        }
+
+        protected override void Go(string[] args)
+        {
+            if (!DateTimeOffset.TryParse(args[0], out DateTimeOffset time))
```

Comment:
The Schedule Time validation looks duplicated. Should be refactored out into a common method.